bucket/retdec.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "4.0",
+    "description": "A retargetable machine-code decompiler based on LLVM.",
+    "homepage": "https://github.com/avast/retdec",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/avast/retdec/blob/master/LICENSE"
+    },
+    "notes": [
+        "It's recommended to install any version of Python >= 3.4",
+        " (for ex. you can type 'scoop install python').",
+        "You could install the following programs:",
+        " - UPX (Optional: if you want to use UPX unpacker in the preprocessing stage);",
+        " - Graphviz (Optional: if you want to generate call or control flow graphs)."
+    ],
+    "depends": "vcredist2022",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/avast/retdec/releases/download/v4.0/retdec-v4.0-windows-64b.7z",
+            "hash": "5fc667da3af3530cf3d393f96627987c3e46da62f4b426e78cdcc25a79416a31"
+        },
+        "32bit": {
+            "url": "https://github.com/avast/retdec/releases/download/v4.0/retdec-v4.0-windows-32b.7z",
+            "hash": "c9b6fc91007c2158209ac730d981a99006a960a27e3b0f54e5618872f34e1afe"
+        }
+    },
+    "extract_dir": "retdec",
+    "env_add_path": "bin",

```

Comment:
Please create an array of .exe files (i.e. `bin` field) instead of `env_add_path` here

Reply:
Ok, done